exercise_03.py

```diff
@@ -0,0 +1,20 @@
+def foot_to_meters(foot):
+    meter = foot * 0.305
```

Comment:
can we just replace `meter` by the value of this variable because it seems it is not required at all.